source/options.css

```diff
@@ -69,15 +69,26 @@ h2 ~ h2 {
 	text-decoration: line-through;
 }
 
+.js-features :not(:checked) + .info .extended-options {
+	display: none;
+}
+
 .js-features .description {
-	font-style: 0.8em;
 	opacity: 0.8;
 }
 
 .js-features a {
 	font-size: 0.9em;
 }
 
+.js-features .info {
+	flex: 1;
```

Comment:
Does this have any effect? If so, can you avoid the shorthand and use the plain property? e.g. `flex-shrink: 1`

Reply:
Yes, the `textarea` doesn't fill entire width if this is not expanded.